agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -228,6 +229,7 @@ class backhaul_manager : public btl::transport_socket_thread {
     bool pending_enable = false;
 
     std::string controller_bridge_mac;
+    bool prplmesh_controller = false;
```

Comment:
Nitpick: I prefer names of booleans to be clearly boolean, so `is_prplmesh_controller` or `controller_is_prplmesh`.